articles/node-date-and-time-objects-with-moment/index.md

```diff
@@ -0,0 +1,531 @@
+
+```js
+const moment = require('moment');
+const myduration =  moment.duration(10);
+console.log(myduration);
+```
+
+By default, moment will return duration in milliseconds, however you can parse unit of measurement 0f your choice as the second argument
+
+```js
+const moment = require('moment');
+const myduration =  moment.duration(10, 'seconds');
+console.log(myduration);
+```
+
+Alternatively, parse the object of values with many units of measurement
+
+```js
+const moment = require('moment');
+const myduration = moment.duration({
+  milliseconds:'29',
+  seconds: '29',
+  minutes: '29',
+  hours: '29',
+  days: '29',
+  weeks: '29',
+  months: '29',
+  years: '29'
+});
+```
+
+**Examples**
+
+Eg1: [Creating a duration clone](https://momentjs.com/docs/#/durations/clone/) (mutable durations) allows you to get a snapshot of time as some points
+
+```js
+const moment = require('moment');
+const duration1 = moment.duration();
+const duration2 = duration1.clone();
+console.log(duration1.add(1, 'second'));
+console.log(duration1.asMilliseconds() !== duration2.asMilliseconds());
+console.log(duration2);
+```
+
+Eg2: [Humanize](https://momentjs.com/docs/#/durations/humanize/) is used to display length of time with duration suffix. Humanize returns a string that describes a duration.
+
+<iframe height="400px" width="100%" src="https://repl.it/@kimkimani/m-durations?lite=true" scrolling="no" frameborder="no" allowtransparency="true" allowfullscreen="true" sandbox="allow-forms allow-pointer-lock allow-popups allow-same-origin allow-scripts allow-modals"></iframe>
+
+[Humanize](https://momentjs.com/docs/#/durations/humanize/) calculates the unit of time parsed and if the unit exceed to the next unit, the next unit will be returned.
+
+I.e. 24 hours will return as day since one day is equivalent to 24 hours. The length of duration in a day is 24 hours. However, if you only need the specified unit to be returned a second argument (UNIT) is parse
+
+**Example**
+
+```js
+const moment = require('moment');
+console.log(moment.duration(27, "hours").asHours());
+```
+### Date queries
+[Moment date queries](https://momentjs.com/docs/#/query/) provide comparison methods such as `isBefore()`, `isAfter()`, `isSame()`, `isBetween()` and `isLeapYear()`. These methods return a Boolean value from the moments comparisons used. If a Moment is not defined, it will return the current time as the default moment.
+
+**Examples**
+
+Eg1: check if a moment [is before](https://momentjs.com/docs/#/query/is-before/) other moments.
+
+```js
+const moment = require('moment');
+console.log(moment('2020-08-12').isBefore('2020-09-20'));
+console.log(moment('2020-08-12').isBefore('2020-07-11'));
+```
+
+Eg2 check if a moment [is same to](https://momentjs.com/docs/#/query/is-same/) another moment.
```

Comment:
Instead of "is same to", use "is same as"

Reply:
Thank you @botahamec